File: C++/problem2.cpp
#include<iostream>
int main(){
    int n;
    std::cin >> n;
    int count10 = 0, count5 = 0, count2 = 0, count1 = 0;
    count10 = n / 10;
    n -= count10 * 10;
    count5 = n / 5;
    n -= count5 * 5;
    count2 = n / 2;
    n -= count2 * 2;
    count1 = n;
    std::cout << count10 << " " << count5 << " " << count2 << " " << count1 << std::endl;
    return 0;
}

File: JS/problem2.js
let n;
	n = parseInt(prompt('Enter a number:'));
let count10 = 0, count5 = 0, count2 = 0, count1 = 0;
	count10 = Math.floor(n / 10);
	n -= count10 * 10;
	count5 = Math.floor(n / 5);
	n -= count5 * 5;
	count2 = Math.floor(n / 2);
	n -= count2 * 2;
	count1 = n;
console.log(count10 + " " + count5 + " " + count2 + " " + count1);